packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx

```diff
@@ -134,11 +132,7 @@ const GroupedSeriesConfiguration: FC<GroupedSeriesConfigurationProps> = ({
     const onDragEnd = useCallback(
         (result: DropResult) => {
             const allSerieIds = series.map(getSeriesId);
-            const seriesWithColor: Series[] = series.map((s, i) => ({
-                ...s,
-                color: getSeriesColor(s, i),
-            }));
-            const serie = seriesWithColor.find(
+            const serie = series.find(
```

Comment:
Grouped series always pick up a consistent color mapping, so this step is no longer doing anything.